frontend/src/components/Box.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from "react";
+
+const Box: FC<
+  React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>
+> = (props) => {
+  return (
+    <div
+      className="m-4 p-2 rounded border-solid border border-primary-500"
```

Comment:
People I have huge respect for always disagree, but I can't help but always find this tailwind quite unpleasant to read. Maybe one I'll just need to get used to!

Reply:
Haha, I totally understand. But at the same time I find this sort of quick development a lot faster when my css and jsx is all in one place. I also find that by having it inline like this, I lean to trying to keep my classes as simplified as possible. Whereas, with separate css files I find classes become really bloated with unnecessary stuff really quickly. Then when you want to update a component, it becomes really difficult to make the change you want.

Reply:
I could definitely be talked into separating out the css down the line when I've got a more stable structure to the frontend. I just hate having to jump around between css files while I'm quickly prototyping.